custom-recipes/api-connect/recipe.json

```diff
@@ -254,6 +281,7 @@
             "name": "ignore_ssl_check",
             "label": "Ignore SSL check",
             "type": "BOOLEAN",
+            "visibilityCondition": "model.auth_type!='secure_oauth' && model.auth_type!='secure_basic'",
```

Comment:
Oups, one boolean has been forgotten here,  could you change this one too? 
<img width="227" alt="Capture d’écran 2024-04-18 à 16 49 27" src="https://github.com/dataiku/dss-plugin-api-connect/assets/77485161/5e917e76-8cc9-4777-9f43-18d216c1bfc0">

Reply:
Fixed in [7bc5186](https://github.com/dataiku/dss-plugin-api-connect/pull/40/commits/7bc51861b69b8161c4328d91c04ca4e5dd917a74)